package.json

```diff
@@ -42,26 +42,66 @@
 			}
 		],
 		"breakpoints": [
-			{"language": "c"},
-			{"language": "cpp"},
-			{"language": "d"},
-			{"language": "objective-c"},
-			{"language": "fortran"},
-			{"language": "fortran-modern"},
-			{"language": "fortran90"},
-			{"language": "fortran_free-form"},
-			{"language": "fortran_fixed-form"},
-			{"language": "rust"},
-			{"language": "pascal"},
-			{"language": "objectpascal"},
-			{"language": "ada"},
-			{"language": "nim"},
-			{"language": "arm"},
-			{"language": "asm"},
-			{"language": "vala"},
-			{"language": "crystal"},
-			{"language": "kotlin"},
-			{"language": "zig"}
+			{
```

Comment:
Can you please amend your commit with removing that formatting change? The one with the dependency re-order is fine.

Reply:
Fixed